tools/report_build.py

```diff
@@ -347,18 +410,19 @@ def main():
             # found a recent commit we can update
             for line in event["node"]["body"].splitlines():
                 # include all builds with a different build_type
-                if not line.startswith(args.build_type):
+                if not line.startswith(f"{args.build_type} "):
```

Comment:
Why the change? What difference does the space make?

Reply:
In case of "Type" and "Type2", "Type" would take "Type2" over when updating the comment.

This is not entirely future-proof, but good enough for the formatting we have now.

Reply:
OK, fair enough